apps/web/src/common/modules/navigations/lnb/modules/LNBRouterMenuItem.vue

```diff
@@ -42,6 +42,10 @@ const isSelectedMenu = (selectedMenuRoute: Location): boolean => {
     if (!resolvedHref.endsWith('/')) resolvedHref += '/';
     return currentPath.startsWith(resolvedHref);
 };
+const getIconName = (icon): string => {
```

Comment:
nitpick: :)

```typescript
const getIconName = (icon: LNBItem['icon']): string => {
```

Reply:
But.. `LNBItem['icon']` can be `undefined`..